docs/getting-started/local-setup/setting-up-a-local-chain.md

```diff
@@ -27,6 +27,17 @@ which is ready to accept transactions and interact with other Gno nodes.
 
 ![gnoland start](../../assets/getting-started/local-setup/setting-up-a-local-chain/gnoland-start.gif)
 
+:::info Lazy init
+
+Starting a Gno blockchain node using just the `gnoland start` command implies a few things:
+
+- the default configuration will be used, and generated on disk in the `gnoland-data` directory
+- random secrets data will be generated (node private keys, networking keys...)
+- an entirely new `genesis.json` will be used, and generated on disk in the `../gnoland-data` directory. The genesis
+  will have a single validator, whose public key is derived from the previously generated node secrets
+
```

Comment:
Nit: use a numbered bullet list, start items with a capital letter, and end the sentence with a comma or period.

Reply:
This doesn't need to be a numbered list, it's purposefully an unordered list

Do we have these docs guidelines outlined anywhere?

Reply:
This is a nitpick - of course it doesn't have to be a numbered list, but for me personally it looks better if it is. If you do not want to apply my suggestion, please resolve this conversation.